tests/console/check_system_info.pm

```diff
@@ -56,11 +58,19 @@ sub check_product {
     $mypro //= uc get_var('SLE_PRODUCT', '');
     my $myver   = get_var('VERSION');
     my $product = '';
-    if ($mypro =~ /HPC/) {
+    if (($mypro eq "HPC") && !get_var("MEDIA_UPGRADE")) {
         $product = 'SLE-Product-HPC-' . $myver;
     }
-    if ($mypro ne '') {
-        $product = $mypro . $myver;
+    if (($mypro eq "SLED") || ($mypro eq "SLES")) {
+        if ((get_var("HDDVERSION") =~ /leap/) && $flag eq '1') {
+            $product = "openSUSE-Leap";
+        } else {
+            $product = $mypro . $myver;
+        }
+    }
+    if ($mypro eq "SLE_HPC") {
+        $product = "SLES" . $myver             if ($flag eq '1');
```

Comment:
I think the flag can be replaced by some version check?

Reply:
It related with before/after migration. So cannot use version check